src/pygambit/game.pxi

```diff
@@ -886,13 +887,13 @@ class Game:
                 raise KeyError(f"{funcname}(): no strategy with label '{strategy}'")
         raise TypeError(f"{funcname}(): {argname} must be Strategy or str, not {strategy.__class__.__name__}")
 
-    def _resolve_node(self, node: typing.Any, funcname: str, argname: str = "node") -> Node:
-        """Resolve an attempt to reference a node of the game.
+    def _resolve_node(self, node: typing.Any, funcname: str, argname: str = "node") -> typing.Iterable[Node]:
```

Comment:
You cannot change the return type of this function, for two reasons:

(1) Sometimes you are returning a node and sometimes an iterable of nodes.  So your implementation does not match the type hints.

(2) This will break other functions that call this, because at the moment we do not support lists of nodes in other functions.  We may do so eventually but for now it is a bug to allow it.